internal/dag/builder_test.go

```diff
@@ -858,8 +859,9 @@ func TestDAGInsert(t *testing.T) {
 						Paths: []networking_v1.HTTPIngressPath{{
 							Backend: *backendv1("kuard", intstr.FromString("http")),
 						}, {
-							Path:    "/kuarder",
-							Backend: *backendv1("kuarder", intstr.FromInt(8080)),
+							Path:     "/kuarder",
+							PathType: (*networking_v1.PathType)(pointer.StringPtr("Exact")),
```

Comment:
The `PathType` field is a `*PathType` and the `PathType` type is an alias for `string`

This is correct and what VSCode gives me as a completion but it is ugly, maybe worth a test helper function